config/environments/production.rb

```diff
@@ -87,7 +87,10 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_mailer.default_url_options = { host: 'trailmix.life' }
+  config.action_mailer.default_url_options = {
+    protocol: 'https',
+    host: 'www.trailmix.life'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.